### PR TITLE
component width

### PR DESCRIPTION
**Pull request by gvonkoss**
@TheoLeanse

### CODE REVIEW

File: src/lib/timestamps.js [hunk 1]
```diff
@@ -1,22 +0,0 @@
-module.exports = function timestamps (data) {
```
Comment (TheoLeanse) [inline src/lib/timestamps.js]: how come we don't need this anymore?

Comment (gvonkoss) [reply src/lib/timestamps.js]: er.

Comment (gvonkoss) [reply src/lib/timestamps.js]: i deleted the wrong file 🙈

**Comment by TheoLeanse** [comment]
+1

**Pull request merged**